### ISSUE TITLE
ros console not working

### ISSUE THREAD

**Issue by rbucker** [rancher/os#2273]
**RancherOS Version: (ros os version)**
1.2.0 running in VMWare and created with docker-machine

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
baremetal VMware ESXi 6.5 (2017-12)

```
for h in {z01,z02}; do \
    docker-machine create \
    --driver vmwarevsphere \
    --vmwarevsphere-disk-size 16384 \
    --vmwarevsphere-memory-size 4096 \
    --vmwarevsphere-boot2docker-url https://github.com/rancher/os/releases/download/v1.1.0/rancheros.iso \
    --vmwarevsphere-username=$vmusr --vmwarevsphere-password=$vmpwd \
    --vmwarevsphere-vcenter 192.168.86.110 $h ;\
    docker-machine ssh $h "echo '#cloud-config' >cloud && echo 'ssh_authorized_keys:' >>cloud && cat .ssh/authorized_keys* >>cloud  && sed -e 's/^ssh-/  - ssh-/g' -i.bak cloud" ;\
    docker-machine ssh $h "sudo ros install -f -c cloud -d /dev/sda -i rancher/os:v1.2.0" ;\
    docker-machine ssh $h "sudo ros os upgrade -f" ;\
    docker-machine regenerate-certs -f $h ;\
    docker-machine ssh $h "sudo ros engine switch docker-17.12.1-ce" ;\ 
    docker-machine ssh $h "sudo ros console switch -f alpine" ;\ 
done 
docker-machine ls
```

No matter what I do ``` sudo ros console switch -f alpine ``` does not change the console. I also tried ``` enable ``` with a reboot and the default remained the same.

```
$ docker-machine ssh vm03
[docker@vm03 ~]$ sudo ros console list                   
disabled alpine
disabled centos
disabled debian
current  default
disabled fedora
disabled ubuntu
```

I would also add the ``` default ``` is a bad console name.

**Comment by niusmallnan** [comment]
I can not reproduce on GCE.

Can you please manually run `ros console switch -f alpine` on your setup?
I want to see if there are some error logs.

**Comment by rbucker** [comment]
Thanks ... before I put together this script I had executed the steps manually and was not able to get the switch to work.... I provided the script as a way to describe my environment. In my initial experience the ```console switch``` prompted me with the ```y/n``` etc... but never executed the switch...

ALL THAT SAID... I just ran the command AGAIN and it worked. So I'm closing the issue.  Thanks again.
